Client/Frontend/Settings/SettingsViewController.swift

```diff
@@ -314,6 +314,15 @@ class SettingsViewController: TableViewController {
         option: Preferences.General.mediaAutoBackgrounding,
         image: UIImage(named: "background_play_settings_icon", in: .current, compatibleWith: nil)!.template),
     ])
+    
+    let privacyReportRow = Row(
```

Comment:
If I understand the purpose of this line correctly, we're adding these redirects to the privacy report?

Redirecting from new reddit to old reddit doesn't seem like a privacy feature to me. It enables the same kind of tracking by Reddit either way. It's more of a (really nice!) convenience feature.

Reply:
bad copy/paste, sorry, not a privacy feature of course